packages/runtime/lib/api.js

```diff
@@ -7,7 +7,6 @@ const { PlatformaticApp } = require('./app')
 class RuntimeApi {
   #services
   #dispatcher
-  #loaderPort
```

Comment:
why was this removed?

Reply:
It's not used. It's not related to this PR.